src/graph/graph.c

```diff
@@ -183,19 +183,20 @@ void Graph_ApplyAllPending(Graph *g) {
 
     for(int i = 0; i < array_len(g->labels); i ++) {
       M = g->labels[i];
-      _Graph_ApplyPending(M);
+      g->SynchronizeMatrix(g, M);
```

Comment:
this means that although caller asked to apply all pending changes, it depends on the sync strategy if we'll really going to flush